frontend/package-lock.json

```diff
@@ -9607,7 +9607,9 @@
       }
     },
     "node_modules/caniuse-lite": {
-      "version": "1.0.30001651",
+      "version": "1.0.30001718",
+      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001718.tgz",
+      "integrity": "sha512-AflseV1ahcSunK53NfEs9gFWgOEmzr0f+kaMFA4xiLZlr9Hzt7HxcSpIFcnNCUkz6R6dWKa54rUz3HUmI3nVcw==",
```

Comment:
Updating the caniuse database is a good thing to do, but if it's done, it should be done in it's own separate PR.

https://ucsb-cs156.github.io/topics/pull_requests/package_lock_json.html